hf_olmo/convert_olmo_to_hf.py

```diff
@@ -308,6 +314,9 @@ def main():
     upload_local_checkpoint(local_checkpoint_dir, args.destination_dir)
 
     print(f"Converted checkpoint saved to {args.destination_dir}")
+    if args.cleanup_local_dir:
```

Comment:
Is there ever a reason not to do this?

Reply:
i guess not. But it looks like the script has changed since I l pr'd this. So let me make sure my script works with it.

Reply:
I removed the if statement.